proxies/nodejs/handlers/location.ts

```diff
@@ -24,37 +25,56 @@ export const handleLocation = async (
             JSON.parse(body.params),
             data
         )
+        const lastParam = params[params.length - 1]
+
+        if (lastParam instanceof URL) {
+            const callbackURL: URL = lastParam
+            const onUpdateCallback = (data) => {
+                axios
+                    .post(callbackURL.href, {
+                        entityType: getEntityFromType(data.constructor.name),
+                        data: data
+                    })
+                    .then((resp: any) => console.log('onUpdatecallback data ', resp.data))
+                    .catch((e) => console.error(e))
+            }
+            invokeCommand(entity, command, params, body.isAsync).then((data) => {
+                onUpdateCallback(data)
+            })
+            ctx.status = 200
+            ctx.body = {
+                entityType: 'pending', // TODO: add pending entity type
```

Comment:
if these don't make sense to have in the return statement we don't need to include it as pending

Reply:
any idea on what should return object be?

Reply:
could just send the logs instead of returning an entitytype and data

Reply:
Logger only exists on the `DVCClient` object, that would limit the usage.

sending the logs does not work as expected, 
```
            console.log('logger', entity.logger.info())
            console.log('logger', entity.logger.info('test'))
            console.log('logger', entity.logger.info)
```
outputs 
```
logger undefined
logger undefined
logger [Function: noOpLog]
```

couldn't find a function call we can make to get the logs..

Reply:
well in either case i don't see a scenario where an object has a param that is a callback but also returns an object